frontend/src/components/TopicCard/Links.js

```diff
@@ -25,7 +25,6 @@ class Links extends Component {
     this.props.updateTopic({
       metacode_id: metacodeId
     })
-    this.props.redrawCanvas()
```

Comment:
Did you test it? I think i added that line because it wasn't working otherwise. But if it worksit works

Although maybe we should leave a comment here for if we switch off of backbone?